src/missed_spans.rs

```diff
@@ -260,34 +260,9 @@ impl<'a> FmtVisitor<'a> {
             self.config.max_width() - self.block_indent.width(),
         );
         let comment_shape = Shape::legacy(comment_width, comment_indent);
-
-        if on_same_line {
```

Comment:
the `on_same_line` variable is used by the version Two code removed above, and so this block should be re-inserted.